build/cqs.tsv

```diff
@@ -68,6 +88,11 @@ it.geosolutions.jaiext.utilities:jt-utilities	1.0.9	compile
 it.geosolutions.jaiext.vectorbin:jt-vectorbin	1.0.9	compile
 it.geosolutions.jaiext.warp:jt-warp	1.0.9	compile
 it.geosolutions.jaiext.zonal:jt-zonal	1.0.9	compile
+javax.activation:activation	1.1	compile
+javax.inject:javax.inject	1	compile
+javax.servlet:servlet-api	2.5	compile
```

Comment:
this is going to be rejected - see https://dev.eclipse.org/ipzilla/show_bug.cgi?id=9610 where it was already looked at. I'm going to try excluding it.